tests/testthat/test-globals.R

```diff
@@ -28,7 +28,8 @@ test_that("parseGlobals works", {
              "#' @apiBasePath basepath",
              "#' @apiSchemes schemes",
              "#' @apiConsumes consumes",
-             "#' @apiProduces produces")
+             "#' @apiProduces produces",
+             "#' @apiTag tag description")
```

Comment:
Multiple tags added in test.
I've also added a test to check for redundant `apiTag` definitions (= duplicates). Since swagger is pretty relaxed about unused tag definitions, I did not add a test to see wether all `apiTag` definitions are also referenced in at least one endpoint.